lib/Runtime/Language/AsmJsLink.cpp

```diff
@@ -143,18 +146,21 @@ namespace Js{
         }
 
 #ifdef ENABLE_SIMDJS
-        Var asmSimdObject = JavascriptOperators::OP_GetProperty(stdlib, PropertyIds::SIMD, scriptContext);
-        for (int i = 0; i < AsmJsSIMDBuiltinFunction::AsmJsSIMDBuiltin_COUNT; i++)
+        if (!simdBuiltinUsed.IsAllClear())
         {
```

Comment:
Change looks good. Just curious, 

>>Since SIMD is not on the global object (this is behind a flag), the property lookup fails and we need to go to the host to complete the lookup. Unfortunately, when this happens in Edge, we mark the property lookup as potentially having side effects and abort asm.js validation (as side effecting operations may cause observable semantic >> differences when running in js vs. asm.js).

So every asm.js validation failed without this change?